src/defs.h

```diff
@@ -163,3 +163,8 @@ typedef struct {
     char alias[MAX_VAR_LEN];
     int value;
 } constant_t;
+
+typedef struct {
+    int index;
+    int next[128];
+} trie_t;
```

Comment:
See https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e